sdl2.vapi

```diff
@@ -1959,16 +1959,16 @@ namespace SDL {
 			public int get_alpha_mod (out uint8 alpha);
 
 			[CCode (cname = "SDL_SetTextureBlendMode")]
-			public int set_blend_mod (Video.BlendMode blend_mode);
+			public int set_blend_mode (Video.BlendMode blend_mode);
 
 			[CCode (cname = "SDL_GetTextureBlendMode")]
-			public int get_blend_mod (out Video.BlendMode blend_mode);
+			public int get_blend_mode (out Video.BlendMode blend_mode);
 
 			[CCode (cname = "SDL_UpdateTexture")]
-			public int update (Video.Rect? rect, void* pixels, int pitch);
+			public int update (Video.Rect? rect, out void* pixels, out int pitch);
```

Comment:
Because I'm an idiot. I saw the same arguments for both functions when I was correcting do_lock and I made an assumption that both needed the fix... I can attest that do_lock does work however since I'm using it in my code.